test/jdk/java/net/httpclient/lib/jdk/httpclient/test/lib/http2/BodyInputStream.java

```diff
@@ -42,7 +42,7 @@ class BodyInputStream extends InputStream {
     final Queue<Http2Frame> q;
     final int streamid;
     boolean closed;
```

Comment:
Maybe `closed` should be volatile as well

Reply:
Agreed, this is the case now